.pre-commit-config.yaml

```diff
@@ -3,4 +3,5 @@ repos:
     rev: stable
     hooks:
     - id: black
-      language_version: python3.7
\ No newline at end of file
+default_language_version:
+  python: python3.7
```

Comment:
I needed to change this because the notebooks environment (for example) now ships with Python 3.8.